multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```
@@ -126,6 +126,9 @@ spec:
   selector:
     app: {{.Values.gateway.name}}
   type: {{ .Values.gateway.serviceType }}
+{{- if .Values.gateway.serviceExternalTrafficPolicy }}
+  externalTrafficPolicy: {{ .Values.gateway.serviceExternalTrafficPolicy }}
+{{- end }}
```

A more succinct way of doing this:
```suggestion
{{- with .Values.gateway.serviceExternalTrafficPolicy }}
  externalTrafficPolicy: {{ . }}
{{- end }}
```

---

Ah thank you, that is indeed a cleaner way! Pushed a new version with this change